packages/web-components/readme.md

```diff
@@ -16,22 +21,19 @@ Stencil components are just Web Components, so they work in any major framework
 
 ## Getting Started
 
-To start building a new web component using Stencil, clone this repo to a new directory:
-
-```bash
-git clone https://github.com/ionic-team/stencil-component-starter.git my-component
-cd my-component
-git remote rm origin
-```
+To start building a new web component using Stencil:
 
-and run:
+run:
 
 ```bash
 npm install
```

Comment:
I think all of the `npm` instructions should be changed to `yarn` instructions.

Reply:
can do :+1: